docs/_static/versions.json

```diff
@@ -1,6 +1,11 @@
 [
     {
-        "name": "v0.9.0 (stable)",
+        "name": "v0.10.0 (stable)",
+        "version": "0.10.0",
+        "url": "https://fairlearn.org/v0.10.0/"
```

Comment:
Similar to @romanlutz 's comment below, I think that this should be `v0.10` not `v0.10.0`

Reply:
To confirm, should I change name, version, and url to 0.10?

Reply:
Comparing to the previous two versions, it should be just the URL. The other fields have the full version.

Reply:
Gotcha, just made the change!

Reply:
I'm seeing the url as `"https://fairlearn.org/v0.10.0/"` not `"https://fairlearn.org/v0.10/"` ?

Reply:
That's odd, I just committed the change again. Do you see it now?